ska_helpers/paths.py

```diff
@@ -15,20 +18,25 @@ def chandra_models_repo_path() -> Path:
     This returns a Path object pointing at the ``chandra_models`` repository in the
     Ska data directory. By default is this returns ``$SKA/data/chandra_models``.
 
-    If the environment variable ``THERMAL_MODELS_DIR_FOR_MATLAB_TOOLS_SW`` is set then
-    the path will be ``$THERMAL_MODELS_DIR_FOR_MATLAB_TOOLS_SW``.
+    The default root can be overridden by setting either of these environment variables,
+    which are checked in the order listed:
+
+    - ``CHANDRA_MODELS_REPO_DIR``
+    - ``THERMAL_MODELS_DIR_FOR_MATLAB_TOOLS_SW``
 
     :returns: Path object
         Path to ``chandra_models`` repository
     """
+    for env_var in CHANDRA_MODELS_ROOT_ENV_VARS:
+        if env_var in os.environ:
+            return Path(os.environ[env_var])
```

Comment:
Feels a little odd to handle this via an "early" `return` but it is a short function.

Reply:
From what I've read early returns are generally a good thing and encouraged to reduce code complexity. Here is an opinion from the internet that must therefore be true. http://www.itamarweiss.com/personal/2018/02/28/return-early-pattern.html

Reply:
It's worth saying that this is something new-ish to me and might not be reflected in earlier code / patterns.